### ISSUE TITLE
[SOLUTION] 4주차 문제 풀이

### ISSUE THREAD

**Issue by myunbongs** [myunbongs/cgvm-winter-study-2023#6]
## 문제 내용
- [ ] [Q1: Multi-Layer Fully Connected Neural Networks](https://cs231n.github.io/assignments2021/assignment2/)
- [ ] [Q2: Batch Normalization](https://cs231n.github.io/assignments2021/assignment2/)
